mlocal/frags/Makefile.stub

```diff
@@ -31,18 +31,21 @@ go_BIN := $(singularity)
 go_OBJ := $(SOURCEDIR)/src/pkg/buildcfg/config.go
 go_INSTALL := $(singularity_INSTALL)
 
+bash_completion :=  $(SOURCEDIR)/etc/bash_completion.d/singularity
+bash_completion_INSTALL := /etc/bash_completion.d/singularity
```

Comment:
As per the discussion, you still need to change bash_completion_INSTALL and prepend it with $(PREFIX).